library/src/main/java/org/apache/apex/malhar/lib/utils/serde/GenericSerde.java

```diff
@@ -34,31 +39,60 @@
 @InterfaceStability.Evolving
 public class GenericSerde<T> implements Serde<T>
 {
-  private transient ThreadLocal<Kryo> kryos = new ThreadLocal<Kryo>()
```

Comment:
https://github.com/EsotericSoftware/kryo says kryo is not thread safe. Are we sure that GenericSerde is always only used by one thread?